.github/workflows/release.yaml

```diff
@@ -423,8 +448,8 @@ jobs:
         uses: ./.ci/build-asset-artifacts
         with:
           dart-ws: ${{ env.DART_WORKSPACE }}
-          wasm-version: ${{ needs.build-release-wasm-libs.outputs.version }}
-          wasm-out-dir-path: ${{ needs.build-release-wasm-libs.outputs.upload-path }}
+          wasm-version: ${{ needs.build-release-wasm-libs.outputs.sequential-version }}
```

Comment:
Is it correct that this assumes the sequential version and the parallel version are the same?

 (Which is ok, but maybe add a comment).

Reply:
this is going to be changed in the follow up pr #202. they are going to be different, but for now the asset generation has not yet been changed, hence the generated assets are the same.